### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by anikarni**

### CODE REVIEW

**Comment by anikarni** [comment]
Not too sure how to deal with this section: https://pixelated-project.org/faq/#relation-to-thoughtworks-and-leap

File: faq.html [hunk 1]
```diff
@@ -171,12 +167,12 @@ <h1 id="how-can-i-keep-myself-safe">How much privacy does Pixelated provide?</h1
...
 
-    <p>We know that it can still be a bit tough to set up your Pixelated environment and find an issue that&#39;s a good fit for your skills, and we&#39;re always looking for ways to lower the entry barrier for new contributors. If you hit any roadblock, you have good ideas, or just want to share your thoughts about onboarding etc., contact our community at <a href="mailto:community@pixelated-project.org" target="_blank">community@pixelated-project.org</a> so we can resolve these issues for you and everyone else. We also have an <a href="irc://irc.freenode.net/pixelated">IRC channel</a>.</p>
+    <p>We know that it can still be a bit tough to set up your Pixelated environment and find an issue that&#39;s a good fit for your skills, and we&#39;re always looking for ways to lower the entry barrier for new contributors. If you hit any roadblock, you have good ideas, or just want to share your thoughts about onboarding etc., contact our community at <a href="mailto:community@pixelated-project.org" target="_blank">community@pixelated-project.org</a> so we can resolve these issues for you and everyone else.
```
Comment (deniscostadsc) [inline faq.html]: Why did you remove the IRC channel, but kept the community mail?

**Review by deniscostadsc — COMMENTED** [review]
The only thing I don't understand is the IRC removal.

File: faq.html (hunk 1 above)
Comment (anikarni) [inline faq.html]: Good question. Mostly because IRC requires activity, which I don't think we can guarantee. We need to decide if there will be any communication channels at all. Thoughts @olabini

File: faq.html [hunk 2]
```diff
@@ -441,10 +437,6 @@ <h1 id="contact-the-project">Contact the project</h1>
       <li>Do you want to report abuse? Send an email to: <a href="mailto:abuse@pixelated-project.org" target="_blank">abuse@pixelated-project.org</a></li>
       <li>Thoughts about onboarding, community management, etc.? Send an email to: <a href="mailto:community@pixelated-project.org" target="_blank">community@pixelated-project.org</a></li>
```
Comment (tuliocasagrande) [inline faq.html]: Same here, community and abuse emails were kept.

**Review by tuliocasagrande — APPROVED** [review]
